Week3/homework/2-step3.js

```diff
@@ -2,7 +2,23 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const values = [];
-  // Add your code here
+  for (startIndex; startIndex <= stopIndex; startIndex++) {
+    values.push(startIndex);
+  }
+  for (const i in values) {
+    if (values[i] % 3 === 0) {
+      threeCallback(values[i]);
+    } else values[i] % 5 === 0;
```

Comment:
it's working but where is the `if` statement?
And what if you have a condition that you will see divisible on 5 and 3 like 15 or 30.